src/ERC20Reserve.sol

```diff
@@ -5,54 +5,49 @@ import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
 import {IERC20} from "openzeppelin-contracts/token/ERC20/IERC20.sol";
 
 interface IERC20Reserve {
-    function erc20() external view returns (IERC20);
+    function withdrawable(IERC20 token) external view returns (uint256 amt);
```

Comment:
I still would prefer to use `address` here. It just results in unnecessary casting which is anyway not enforced.
We had the discussion already. :)

Reply:
We can keep it.